izadji/src/routes.ts

```diff
@@ -9,7 +9,8 @@ export const routes = {
   project: '/project/:id',
   aboutUs: '/about-us',
   donate: '/donate',
-  publication: '/publications'
+  publication: '/publications',
+  documentatio: '/documentation'
```

Comment:
Think you have a typo here, missing letter **n**.

Reply:
fixed it, thanks for catching this.